client/app/app.scss

```diff
@@ -143,3 +143,7 @@ h2, h3, h4, h5, h6 {
   line-height: 15.3999996185303px;
   margin-top: 5px;
 }
+
+.nav-ticker {
+  color: white;
+}
```

Comment:
Just added one in, not sure if it's displaying on GitHub though...locally, it ends on a newline at 150.